code/modules/assembly/health.dm

```diff
@@ -5,13 +5,13 @@
 	materials = list(MAT_METAL=800, MAT_GLASS=200)
 	attachable = TRUE
 
-	var/scanning = TRUE
+	var/scanning = FALSE
 	var/health_scan
 	var/alarm_health = HEALTH_THRESHOLD_CRIT
 
 /obj/item/assembly/health/examine(mob/user)
 	. = ..()
-	. += "<span class='notice'>Use a multitool to swap between \"detect death\" mode and \"detect critical state\" mode.</span>"
+	. += "<span class='notice'>Use in hand to turn it off/on and Alt-click to swap between \"detect death\" mode and \"detect critical state\" mode.  [src.scanning ? "The sensor is on and you can see [health_scan] displayed on the screen." : "The sensor is off" ]</span>"
```

Comment:
Could you put the current state on a separate line?

Does this read correctly (not just "can see displayed") if the health scanner has not been activated?

Missing a `.` in one of the branches.